e2e/src/DappList.spec.js

```diff
@@ -0,0 +1,14 @@
+import DappListRecent from './usecases/DappListRecent'
+import { quickOnboarding } from './utils/utils'
+import { launchApp } from './utils/retries'
+
+describe('Dapp List', () => {
+  beforeAll(async () => {
+    await quickOnboarding()
+    // Relaunch app to ensure dapp list loads
+    // TODO: Remove this once we have a better way to ensure dapp list loads
+    await launchApp({ newInstance: true })
```

Comment:
the action to fetch the dapps list is called for the first time when the app sagas are kicked off, so it seems like relaunching the app shouldn't add any value in ensuring the dapp list loads? could we either remove this or add some details to the comment about why this helps?

Reply:
Added a comment in 1bcfb9348276381412d0cbc345d7cf59116b3c44, but this is still needed for the tests to pass on the first run.